File: in_go/src/pattern_18.go
package src

import "fmt"

func Pattern_18() {
	row := 5
	column := 9
	symbol := "*"

	for i := 0; i < row; i++ {
		for j := 0; j < column; j++ {
			if j >= i && j <= column-1-i {
				fmt.Print(symbol)
			} else {
				fmt.Print(" ")
			}
		}
		fmt.Println()
	}
}

File: in_java/Pattern_18.java
package in_java;

public class Pattern_18 extends Pattern {
    public Pattern_18(int row, int column, String symbol) {
        super(row, column, symbol);
    }

    public void print() {
        for (int i = 0; i < this.row; i++) {
            for (int j = 0; j < this.column; j++) {
                if (j >= i && j <= column - 1 - i) {
                    System.out.print(this.symbol);
                } else {
                    System.out.print(" ");
                }
            }
            System.out.println();
        }
    }
}
